### DISCUSSION TITLE
블랜더에서 리깅 작업 한 것을 플레이어에게 입힐 수 있는 방법이 있나요?

### DISCUSSION THREAD

**Discussion by ljyfm1** [naverz/zepeto-studio-kor#521] [5. Q&A]
블랜더에서 포즈를 만들고 이것을 어떤 파일로 옮겨야 유니티 제페토에서 포즈 잡기가 가능할까요?

그리고 블랜더에서 포즈를 지정했을 때, 포토존에 도착하면 상호작용 할 수 있는 버튼이 뜨면서 지정된 포즈를 포토존 앞에서 취할 수 있는 기능이 현재 지원되나요??
![Blender_  C__Users_User_Desktop_Zepeto_Unity_pose_pose blend  2022-03-23 오후 1_25_01](https://user-images.githubusercontent.com/102196727/159623786-ae071ff7-effd-4c3d-a401-18f44821bff6.png)

**Comment by x4cf** [comment]
이 자료들이 도움이 될 수 있을까 싶어서 링크걸어보아요 :D 

캐릭터 애니메이션 파일 제작
https://github.com/naverz/zepeto-studio-kor/discussions/154
캐릭터 애니메이션 실행
https://github.com/naverz/zepeto-studio-kor/discussions/151
상호작용 버튼
https://github.com/naverz/zepeto-studio-kor/discussions/157
